src/main/steps/respondent/international-factors/summary/content.ts

```diff
@@ -57,7 +57,7 @@ const en = (content: CommonContent) => {
   return {
     ...enContent,
     language: content.language,
-    sections: [summaryList(enContent, userCase, urls)],
+    sections: [summaryList(enContent, userCase, urls, 'International elements', fieldType, content.language)],
```

Comment:
Can we pass variable 'enContent.title2' as 3rd parameter which we are already defining above.

Reply:
There is 's' in the end for International elements

Reply:
code is updated with 'enContent.title2' as 3rd parameter